### ISSUE TITLE
Rule proposal: `prefer-string-pad-start-end`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1773]
### Description

More clear intention, prevent mistake like this

```text
bar = '12345';
'0'.repeat(4 - bar.length) + bar;
VM433:2 Uncaught RangeError: Invalid count value
    at String.repeat (<anonymous>)

bar = '12345';
bar.padStart(4, '0')
'12345'
```

### Fail

```js
const foo = ' '.repeat(10 - bar.length) + bar;
```

```js
const foo = '*'.repeat(10 - bar.length) + bar;
```

```js
const foo = bar + ' '.repeat(10 - bar.length);
```

```js
const foo = bar + '*'.repeat(10 - bar.length);
```

### Pass

```js
const foo = bar.padStart(10)
```

```js
const foo = bar.padStart(10, '*')
```

```js
const foo = bar.padEnd(10)
```

```js
const foo = bar.padEnd(10, '*')
```

**Comment by sindresorhus** [comment]
Accepted
